ams_flutter/lib/features/onboarding/presentation/pages/signup.dart

```diff
@@ -0,0 +1,224 @@
+    return MaterialApp(
+        debugShowCheckedModeBanner: false,
+        home: Scaffold(
+          appBar: AppBar(
+            title: Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Image.asset(
+                  'Assets/images/img.png',
+                  height: 48,
+                  width: 48,
+                )
+              ],
+            ),
+          ),
+          body: SingleChildScrollView(
+            child: Padding(
+              padding: const EdgeInsets.all(20.0),
+              child: Column(
+                children: [
+                  SizedBox(height: 50),
+                  Padding(
+                    padding: const EdgeInsets.only(right: 110),
+                    child: Text(
+                      "Good to see you!",
+                      style:
+                          TextStyle(fontSize: 32, fontWeight: FontWeight.w700),
+                    ),
+                  ),
+                  SizedBox(height: 15),
+                  Padding(
+                    padding: const EdgeInsets.all(8.0),
+                    child: TextFormField(
+                      decoration: InputDecoration(
+                          labelText: 'Email Address',
+                          border: OutlineInputBorder(
+                            borderRadius: const BorderRadius.all(
+                              const Radius.circular(5.0),
+                            ),
+                          ),
+                          contentPadding: EdgeInsets.symmetric(vertical: 10.0)),
+                    ),
+                  ),
+                  SizedBox(height: 10),
+                  Padding(
+                    padding: const EdgeInsets.all(8.0),
+                    child: TextField(
+                      obscureText: _obscureText,
+                      decoration: InputDecoration(
+                          labelText: 'Password',
+                          border: OutlineInputBorder(
+                            borderRadius: const BorderRadius.all(
+                              const Radius.circular(5.0),
+                            ),
+                          ),
+                          contentPadding: EdgeInsets.symmetric(vertical: 10.0),
+                          suffixIcon: IconButton(
+                            icon: Icon(_obscureText
+                                ? Icons.visibility
+                                : Icons.visibility_off),
+                            onPressed: () {
+                              setState(() {
+                                _obscureText = !_obscureText;
+                              });
+                            },
+                          )),
+                    ),
+                  ),
+                  Row(
+                    children: [
+                      Checkbox(
+                          side: BorderSide(
+                            color: Colors.grey,
```

Comment:
There is a file named `app_colors.dart` in `core/constants/` directory. It contains a class named `AppColor`. 
- If the color exists, please use that variable here.
- Otherwise, add a variable there, and then use it here

This is a form of refactoring, which we had discussed in the previous meet